tmt/steps/execute/run.sh

```diff
@@ -154,6 +154,9 @@ tmt_run_test () {
             tmt_verbose 2 "$tmt_LOGCODE_F:"
             cat "$tmt_LOGCODE_F"
 
+            grep -q '^124$' "$tmt_LOGCODE_F" &&
+            tmt_verbose 2 "Test duration exceeded."
```

Comment:
I think you're missing `\`, also I'd rather format it loke so:

```suggestion
            grep -q '^124$' "$tmt_LOGCODE_F" \
              && tmt_verbose 2 "Test duration exceeded."
```

_I'm not saying it won't work (not sure it does though), but this seems more resilient to me._

Reply:
Ok, modified.